packages/docs/docs/fhir-datastore/patient-deduplication/patient-deduplication.md

```diff
@@ -70,9 +70,9 @@ The Medplum team recommends starting with a batch pipeline. If your source syste
 
 ### Batch Pipelines
 
-Batch pipelines run as offline jobs that considers all records at once to produce pairs of patient matches. As this is an N<sup>2</sup> problem, they are primarily constrained by memory rather than latency.
+Batch pipelines run as offline jobs that consider all records at once to produce pairs of patient matches. Most implementations schedule these pipelines to run on a regularly scheduled interval. As this is an N<sup>2</sup> problem, they are primarily constrained by memory rather than latency.
```

Comment:
Semantics but I don't believe all batch pipelines will "produce **pairs** of patient matches", and could be groups, sets, or pairs?

Reply:
Actually this is a really good clarification. 

Most pipelines first produce pairs, and then those pairs are aggregated in a separate step into groups (aka clusters).

Will add